sdk/include/reactos/wine/test.h

```diff
@@ -155,6 +155,8 @@ extern void __winetest_cdecl winetest_trace( const char *msg, ... );
 #define todo_wine_if(is_todo)   todo_if((is_todo) && !strcmp(winetest_platform, "wine"))
 #endif
 
+#define flaky_skip              win_skip("Flaky test\n"), 0 &&
```

Comment:
We'd really want this to be `ros_skip`, right? I'm guessing none of the flaky failures happen on Windows.

Reply:
> 
> 
> How the user can disable `flaky_skip`?

Currently there is no mechanism for that. What do you think such a mechanism would be useful for? After fixing a test, one would simply remove the corresponding skip and run tests on it a number of times. Enabling all of them at once will probably not help identifying any changes anyway, except when there is only a single flaky test in a file, because otherwise you don't know what test might have failed when.